src/chrome/content/rules/BodyBuilding.com.xml

```diff
@@ -1,73 +1,38 @@
-<!--
-	CDN buckets:
-
-		- d1qro4ibhgqt8c.cloudfront.net 
-			- imagecdn.bodybuilding.com
-
-	bodybuilding.com.112.2o7.net
-
-
-	Nonfunctional domains:
-
-		- bodyspace.bodybuilding.com *
-		- forum.bodybuilding.com
-		- my.bodybuilding.com		(times out)
-
-	* Redirects to http
-
-
-	Partially covered bodybuilding.com subdomains:
-
-		- (www.) *
-		- r[eosu] *
-		- s[abcegikmnortvz] *
-		- t[chjltwz] *
-		- u[ayz] *
-		- v[cgnu] *
-		- ws *
-		- ye *
-		- z[aw] *
-
-	*  Some (most?) pages redirect to http
-
-
-	Fully covered domains:
-
-		- common.bbcomcdn.com
-		- store.bbcomcdn.com
-
-		- bodybuilding.com subdomains:
-
-			- ad
-			- assets
-			- imagecdn	(→ d1qro4ibhgqt8c.cloudfront.net)
-			- n		(→ bodybuilding-com.112.2o7.net)
-			- s
-
--->
-<ruleset name="BodyBuilding.com (partial)">
-
-	<target host="*.bbcomcdn.com" />
+<ruleset name="Bodybuilding.com">
+	<!--
+		*.bbcomcdn.com
+
+		Invalid certificate:
+			- trackers.bbcomcdn.com
+	-->
+	<target host="artifacts.bbcomcdn.com" />
+	<target host="common.bbcomcdn.com" />
+	<target host="store.bbcomcdn.com" />
+
+	<!--
+		*.bodybuilding.com
+
+		Invalid certificate:
+			- cschat.bodybuilding.com
+			- n.bodybuilding.com: equivalent to bodybuilding-com.112.2o7.net
+	-->
+	<target host="bodybuilding.com" />
 	<target host="*.bodybuilding.com" />
```

Comment:
Is there a specific reason for a wildcard here?

Reply:
Not that I can remember, going to enumerate subdomains now.

Reply:
Fixed.